project/plugins.sbt

```diff
@@ -13,3 +13,5 @@ addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.5.2")
```

Comment:
We shouldn't have the scala fix in our plugins after we're done with the upgrade